addons/website_sale/views/templates.xml

```diff
@@ -37,6 +37,7 @@
     <template id="template_header_hamburger" inherit_id="website.template_header_hamburger">
         <xpath expr="//t[@t-foreach='website.menu_id.child_id']" position="after">
             <t t-call="website_sale.header_cart_link">
+                <t t-set="_icon" t-value="True"/>
```

Comment:
The code is clearly showing that the templates are designed to either show the icon (fa-xx) or a text instead ('My Cart').
If you pass the value to `True`, the icon is indeed visible, but that's breaking the layout that was trying to be achieved: Some templates are showing the text, other are showing the icon.